PWGLF/Tasks/Resonances/kstarpbpbsa.cxx

```diff
@@ -0,0 +1,286 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+// K* spin alignment
+// Prottay Das (prottay.das@cern.ch)
+
+#include <TH1F.h>
+#include <TDirectory.h>
+#include <THn.h>
+#include <TLorentzVector.h>
+#include <TMath.h>
+#include <TObjArray.h>
+#include <TFile.h>
+#include <TH2F.h>
+#include <TLorentzVector.h>
+#include <TPDGCode.h>
+#include <string>
+#include <vector>
+#include <cmath>
+#include <array>
+#include <cstdlib>
```

Comment:
You are including plenty of headers while some are not needed, others are included multiple times. Please make sure that you include what you use.